composite/synth-index/src/config.ts

```diff
@@ -0,0 +1,13 @@
+import { util } from '@chainlink/ea-bootstrap'
+
+export type Config = {
+  defaultNetwork: string
+  dataProvider: string
+}
+
+export const makeConfig = (dataProvider = ''): Config => {
+  return {
+    defaultNetwork: util.getEnv('DEFAULT_NETWORK') || 'mainnet',
+    dataProvider,
+  }
+}
```

Comment:
You should create the required underlying config here, something like this:

```suggestion
export const makeConfig = (prefix = ''): Config => {
  return {
    defaultNetwork: util.getEnv('DEFAULT_NETWORK', prefix) || 'mainnet',
    taConfig: ta.makeConfig(prefix)
  }
}
```